src/app/components/search/search-input-control/search-input-control.component.scss

```diff
@@ -1,8 +1,50 @@
-.adf-clear-search-icon-wrapper {
-  width: 1em;
+$top-margin: 12px;
+
+.app-search-container {
+  margin-top: -$top-margin;
+  padding-top: 2px;
+  font-size: 16px;
+
+  .mat-form-field-underline {
+    display: none;
+  }
+
+  .mat-form-field-label-wrapper {
+    cursor: text;
+    pointer-events: auto;
+  }
+
+  .app-input-form-field {
+    letter-spacing: -0.7px;
+    width: 90%;
+
+    .mat-input-element {
+      letter-spacing: -0.7px;
+    }
+  }
+
+  .mat-focused .placeholder {
+    color: transparent;
+  }
+
+  .app-search-button.mat-icon-button {
+    left: -8px;
+    margin-left: 15px;
+
+    .mat-icon {
+      font-size: 24px;
+      padding-right: 0;
+    }
+  }
+}
+
+.app-prefix-search-icon-wrapper {
+  height: 6px;
+  margin: 14px 13px;
+  float: left;
```

Comment:
why do we need a float? :)

Reply:
I had trouble styling that item (might be related to [this issue](https://github.com/angular/material2/issues/10093) - even though it's closed) and float is the thing that fixed it... :)